src/test/ui/issues/issue-5239-1.stderr

```diff
@@ -5,11 +5,6 @@ LL |     let x = |ref x: isize| { x += 1; };
    |                              -^^^^^
    |                              |
    |                              cannot use `+=` on type `&isize`
-   |
-help: `+=` can be used on `isize`, you can dereference `x`
-   |
-LL |     let x = |ref x: isize| { *x += 1; };
```

Comment:
This is kinda a regression, but not actually... `&isize` _doesn't_ have a `+=`. Perhaps we could suggest changing the ref to `ref mut`, but that seems out of scope for this PR.